chromium/background.js

```diff
@@ -60,6 +60,10 @@ chrome.storage.onChanged.addListener(async function(changes, areaName) {
       isExtensionEnabled = changes.globalEnabled.newValue;
       updateState();
     }
+    if ('debugging_rulesets' in changes) {
+      Object.assign(all_rules, new rules.RuleSets());
+      await all_rules.initialize();
```

Comment:
It might be better to do:
```javascript
r = new rules.RuleSets();
await r.initialize();
Object.assign(all_rules, r);
```

Otherwise, `all_rules` will temporarily be populated with empty data.

Reply:
Good call. Changing now.